### PR TITLE
Feature/token auth

### PR DESCRIPTION
**Pull request by bewakes**

### CODE REVIEW

File: core/tasks/get_data.py [hunk 1]
```diff
@@ -129,8 +153,11 @@ def fetch_project_data(
 
         logger.info(f"Fetched all data for deep project {pid}")
         print(f"Fetched all data for deep project {pid}")
```
Comment (ranjan-stha) [inline core/tasks/get_data.py]: print statement.

File: core/tasks/queries.py [hunk 2]
```diff
@@ -87,6 +87,16 @@
...
 """
 
+new_projects_q = """
+select pp.id, pp.status from project_project pp
+    left join lead_lead ll
+    on ll.project_id=pp.id
+where pp.is_private=false
+    and pp.is_deleted=false
+group by pp.id
+having count(ll.id) > 10
```
Comment (ranjan-stha) [inline core/tasks/queries.py:97]: I guess greater than 10 is quite random, right?

Comment (bewakes) [reply core/tasks/queries.py:97]: Yes dai, filtering by projects which have more than 10 leads reduces the number of projects to fetch by a lot. I'll leave a comment there.

File: core/tasks/get_data.py (hunk 1 above)
Comment (bewakes) [reply core/tasks/get_data.py]: OOps. Fixed.

**Review by ranjan-stha — APPROVED** [review]

**Pull request merged**